apps/site/components/EOL/UnknownSeveritySection/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import VulnerabilitiesTable from '#site/components/EOL/VulnerabilitiesTable';
+import type { Vulnerability } from '#site/types/vulnerabilities';
+
+const UnknownSeveritySection: FC<{
```

Comment:
I still believe this compoonent should receive ther whole list of vulnerabilities and itself filter/map the unknown ones.

Since you're not tightenining the types of `vulnerabilities: Array<Vulnerability>;` nothing prevents me to inadvertedly pass any sort of vulnerability. Either tighten the types here to be a subset of Vulnerability where severity is unknown or pass all vulns and let the component maintain itself. Otherwise you're creating external dependencies + need of duplicating code.

Reply:
addressed via https://github.com/nodejs/nodejs.org/pull/7990/commits/4836befb035e54734f30fb9037056043f5fc22dd